runtime/doc/news.txt

```diff
@@ -189,6 +189,8 @@ The following new APIs and features were added.
 
 • Added |:fclose| command.
 
+• Added |vim.snippet| for snippet expansion support.
```

Comment:
I mentioned this in its own bullet instead of putting it under LSP, but lemme know if you want to change that.